products/sle12/profiles/pci-dss.profile

```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+metadata:
+    SMEs:
+        - abergmann
+
+reference: https://docs-prv.pcisecuritystandards.org/PCI%20DSS/Standard/PCI-DSS-v4_0.pdf
```

Comment:
3.2.1 instead?

Reply:
Should be ok in [0e0c5db](https://github.com/ComplianceAsCode/content/pull/9613/commits/0e0c5db0fe96c5e52f8bf788041cd8350d629baa)